MapboxDirectionsTests/MatchTests.swift

```diff
@@ -0,0 +1,103 @@
+import XCTest
+import OHHTTPStubs
+@testable import MapboxDirections
+
+class MatchTests: XCTestCase {
+    override func tearDown() {
+        OHHTTPStubs.removeAllStubs()
+        super.tearDown()
+    }
+    
+    func test() {
```

Comment:
I know it's punctilious, but could we have a more descriptive name for this test? maybe `testDirectionsMatchingResponseParsing`?